clearwater-etcd/usr/bin/clearwater-etcdctl

```diff
@@ -15,4 +15,4 @@ if [ -z "$target_ip" ]; then
 fi
 
 # Run the real etcdctl.
-/usr/share/clearwater/clearwater-etcd/$etcd_version/etcdctl -C $target_ip:4000 "$@"
+/usr/share/clearwater/clearwater-etcd/$etcd_version/etcdctl -C http://$target_ip:4000 "$@"
```

Comment:
Why does this need to change? Does it work with all versions? I assume you've tested the 3.x ones - I wonder if we should be just removing the 2.x one entirely.

Reply:
I'm not 100% sure, but I assume it's related to the following section from the [upgrade documentation](https://coreos.com/etcd/docs/latest/upgrades/upgrade_3_2.html)

> Change in --listen-peer-urls and --listen-client-urls
> 3.2 now rejects domains names for --listen-peer-urls and --listen-client-urls (3.1 only prints out warnings), since domain name is invalid for network interface binding. Make sure that those URLs are properly formated as scheme://IP:port.

I've checked it works against 3.1.7, but I couldn't be bothered to check with 2.2.5 (partly because I was pretty sure it would do and partly because I wasn't sure why I cared). Any idea what the last CC version we shipped with 2.2.5 was? V10?

Reply:
As discussed, I'm going to remove support for 2.2.5